fp.js

```diff
@@ -18,17 +18,25 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
+people.filter(person => {
   if(person.age < 35){
     peopleYoungerThan35.push(person)
   }
 })
```

Comment:
`filter` should also have a return statement, although this time it should return the condition that you would like each item in the new array to meet. So in the case of creating an array of people younger than 35...

```js
const youngerThan35 = people.filter(person => {
  return person.age < 35;
})
```